### DISCUSSION TITLE
Ubuntu 16.04.7 cannot make

### DISCUSSION THREAD

**Discussion by colorain549** [wfrest/wfrest#202] [Q&A]
When I execute **make** on command line, it shows the error message, as follow, 

CMake Error at CMakeLists.txt:16 (message): 
Workflow is neither installed not in third_party!

Ps
My OS is Ubuntu 16.04.7 and I have already installed workflow, 
Could you please tell me how to fix it, 
thx so much !

**Comment by chanchann** [comment]
Did you use command？git clone --recursive https://github.com/wfrest/wfrest
